tests/Features/ReturnTypes/AuthExtension.php

```diff
@@ -9,7 +9,7 @@
 
 class AuthExtension
 {
-    public function testUser(): User
+    public function testUser(): ?User
```

Comment:
This isn't a proper regression test, but I'm not familiar with the testing strategy used here and haven't had a chance to dig into it.